src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -169,16 +159,14 @@ protected void readFromNBT(NBTTagCompound compound)
 
         //  Workload
         workManager.readFromNBT(compound.getCompoundTag(TAG_WORK));
-
-        //autoHostile = compound.getInteger(TAG_AUTO_HOSTILE);
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)